.github/workflows/flake8.yml

```diff
@@ -0,0 +1,31 @@
+name: flake8 
+
+on:
+  push:
+    paths:
+      - '*.py'
+  pull_request:
+    paths:
+      - '*.py'
```

Comment:
I'm a bit surprised that the actions did not run for this Pull Request, because usually they do.
Could it be that `*.py` is too restrictive (only match `*.py` files in the root) and it should be `**.py` instead?
See the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files) for reference.

Reply:
Updated to `**.p` but I still do not see the workflows running. Maybe that is because the workflow is introduced last thus no commit afterwards matches the conditions?